Sources/NIO/Bootstrap.swift

```diff
@@ -1016,38 +1016,42 @@ public struct ShorthandServerBootstrapOption {
         data.applyOption(to: serverBootstrap)
     }
     
-    private enum ShorthandServerOption {
-        case ReuseAddr(Bool)
-        case AutoRead(Bool)
-        case Backlog(Int32)
+    fileprivate enum ShorthandServerOption {
+        case reuseAddr(Bool)
+        case autoRead(Bool)
+        case backlog(Int32)
         
         func applyOption(to serverBootstrap: ServerBootstrap) {
             switch self {
-                case .AutoRead(let value):
-                    let _ = serverBootstrap.serverChannelOption(ChannelOptions.autoRead, value: value)
-                case .ReuseAddr(let value):
-                    let _ = serverBootstrap.serverChannelOption(ChannelOptions.socketOption(.reuseaddr), value: value ? 1 : 0)
-                case .Backlog(let value):
-                    let _ = serverBootstrap.serverChannelOption(ChannelOptions.backlog, value: value)
+            case .autoRead(let value):
+                _ = serverBootstrap.serverChannelOption(ChannelOptions.autoRead, value: value)
+            case .reuseAddr(let value):
+                _ = serverBootstrap.serverChannelOption(ChannelOptions.socketOption(.reuseaddr), value: value ? 1 : 0)
+            case .backlog(let value):
+                _ = serverBootstrap.serverChannelOption(ChannelOptions.backlog, value: value)
             }
         }
     }
 }
 
+// Equatable and Hashable for the convenience of users.
+extension ShorthandServerBootstrapOption : Equatable, Hashable {}
```

Comment:
```suggestion
extension ShorthandServerBootstrapOption: Hashable {}
```

Mild cleanliness update: `Hashable` implies `Equatable`.

Reply:
Removed surplus Equatable.

Reply:
Done.